core/domain/cron_services.py

```diff
@@ -26,16 +26,27 @@
 MYPY = False
 if MYPY:  # pragma: no cover
     from mypy_imports import base_models
+    from mypy_imports import beam_job_models
     from mypy_imports import datastore_services
+    from mypy_imports import job_models
     from mypy_imports import user_models
 
-(base_models, job_models, user_models) = models.Registry.import_models([
-    models.Names.BASE_MODEL, models.Names.JOB, models.Names.USER])
+(
+    base_models, beam_job_models,
+    job_models, user_models
+) = models.Registry.import_models([
+    models.Names.BASE_MODEL, models.Names.BEAM_JOB,
+    models.Names.JOB, models.Names.USER
+])
 datastore_services = models.Registry.import_datastore_services()
 
 # Only non-versioned models should be included in this list. Activities that
 # use versioned models should have their own delete functions.
-MODEL_CLASSES_TO_MARK_AS_DELETED = (user_models.UserQueryModel,)
+MODEL_CLASSES_TO_MARK_AS_DELETED = {
+    user_models.UserQueryModel: datetime.timedelta(weeks=8),
+    beam_job_models.BeamJobRunModel: datetime.timedelta(weeks=24),
```

Comment:
Small nit, but 6 months is technically 26 weeks.

Reply:
Yeah, I think we should change this in our policies to say weeks straight away as I think it is a bit more precise than months.

Reply:
Let's go with 180 days. This is the standard we generally use.

Reply:
Done.